### PR TITLE
Add Health Check and Bump Operator-SDK to 1.7.2

### PR DESCRIPTION
**Pull request by abdulhaseeb2**

### CODE REVIEW

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-49-e0cd1a1d`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-49-291a36ba`

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -22,26 +22,35 @@ spec:
...
           requests:
             cpu: 100m
             memory: 20Mi
-        env:
-          - name: WATCH_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-          - name: CONFIG_SECRET_NAME
-            value: slack-secret
```
Comment (ahmedwaleedmalik) [inline config/manager/manager.yaml:45]: We need this

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]

**Review by ahmedwaleedmalik — APPROVED** [review]

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-49-914b15ef`

**Pull request merged**